pylint/checkers/exceptions.py

```diff
@@ -201,7 +200,7 @@ def visit_name(self, node: nodes.Name) -> None:
             self._checker.add_message(
                 "notimplemented-raised", node=self._node, confidence=HIGH
             )
-        elif node.name in OVERGENERAL_EXCEPTIONS:
+        elif node.name in self._checker.linter.config.overgeneral_exceptions:
```

Comment:
```suggestion
        elif node.qname in self._checker.linter.config.overgeneral_exceptions:
```
Would this work ? If not we need to infer like in:
```python
                try:
                    exceptions = list(_annotated_unpack_infer(handler.type))
                except astroid.InferenceError:
                    continue
```

Reply:
Nope because `node` is a `Name` node that doesn't have a `qname` attribute. Using the inference as you've shown seems to have worked, thanks.